caexporter.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"context"
+	"net/http"
+	"regexp"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/rs/zerolog/log"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var (
+	up = prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "up",
+		Help: "is caexporter up",
+	})
+
+	lastActivity = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "cluster_autoscaler_last_activity",
+			Help: "LastProbeTime as reported in cluster-autoscaler-status configmap",
+		}, []string{"activity"},
+	)
+)
+
+type Collector struct {
+	kubeClient *kubernetes.Clientset
+	regex      *regexp.Regexp
+}
+
+func newCollector() *Collector {
+	cfg, err := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
+		clientcmd.NewDefaultClientConfigLoadingRules(),
+		&clientcmd.ConfigOverrides{},
+	).ClientConfig()
+	if err != nil {
+		log.Fatal().Err(err).Msg("")
+	}
+
+	log.Info().Msg("creating kubernetes clientset")
+
+	clientset, err := kubernetes.NewForConfig(cfg)
+	if err != nil {
+		log.Fatal().Err(err).Msg("")
+	}
+
+	rx := regexp.MustCompile(`(?s)LastProbeTime:\s*(.{39})`)
+
+	c := &Collector{
+		kubeClient: clientset,
+		regex:      rx,
+	}
+
+	return c
+}
+
+func main() {
+	r := prometheus.NewRegistry()
+	r.MustRegister(newCollector())
+
+	http.Handle("/metrics", promhttp.HandlerFor(r, promhttp.HandlerOpts{}))
```

Comment:
```suggestion
	http.Handle("/metrics", promhttp.Handler())
```

Add also handler for root `/` which print the info about `/metrics` path